javav2/usecases/Creating_serverless_aurora_app/Readme.md

```diff
@@ -7,11 +16,11 @@ You can develop a dynamic web application that tracks and reports on work items
 + Amazon Serverless Amazon Aurora database
 + Amazon Simple Email Service (the AWS SDK for Java SDK version 2 is used to access Amazon SES)
```

Comment:
Amazon Simple Email Service (Amazon SES). (The AWS SDK for Java (v2) is used to access Amazon SES.)

Reply:
fixed